frontend/src/utils/Window.js

```diff
@@ -25,10 +39,40 @@ export function usePreventZoom(scrollCheck = true, keyboardCheck = true) {
 
     document.addEventListener('keydown', handleKeydown);
     document.addEventListener('wheel', handleWheel, { passive: false });
+    document.addEventListener('touchstart', handleTouchStart, {
+      passive: false
+    });
+    document.addEventListener('touchmove', handleTouchMove, { passive: false });
 
     return () => {
       document.removeEventListener('keydown', handleKeydown);
       document.removeEventListener('wheel', handleWheel);
+      document.removeEventListener('touchstart', handleTouchStart);
+      document.removeEventListener('touchmove', handleTouchMove);
     };
   }, [scrollCheck, keyboardCheck]);
 }
+
+export const useLockScroll = (activeString = '', lockedString = 'Canvas') => {
+  const shouldLockScroll = activeString === lockedString || 'NFTs';
```

Comment:
Not sure I understand this, could you give a brief description on why we do/don't lock scrolling?

Reply:
Why do we allow locking of scrolling?
-On mobile devices, currently, pinching to zoom in or out just zoom out/in the whole screen. Locking the body scroll allows us to enable pinch-to-zoom the canvas on mobile.
-On the NFT page, there is a flow that allows minting NFTs. Locking the scroll allows us to select the area we want to mint on mobile. I think I showed that flow in the video.

Also locking of scrolling doesnt have any effect of desktop is just make the mobile experience seamless.